tools/tests/tools/py2fgen/wrappers/references/diffusion/diffusion.py

```diff
@@ -49,6 +55,7 @@ def diffusion_run_wrapper(
     linit,
     on_gpu,
 ):
+    runtime_config.HOOK_BINDINGS_FUNCTION_ENTER("diffusion_run")
```

Comment:
Ideas discussed offline
```suggestion
    runtime_config.HOOK_BINDINGS_FUNCTION_ENTER["diffusion_run"]()
```
```suggestion
    runtime_config.call_hook("enter_diffusion_run")
    runtime_config.call_hook(ENTER, "diffusion_run")
```